src/clj/orcpub/email.clj

```diff
@@ -4,6 +4,11 @@
             [environ.core :as environ]
             [orcpub.route-map :as routes]))
 
+(defn get-http-protocol [] ()
```

Comment:
Unnecessary if we are keeping the `base-url` argument to the `send-verification-email` function